### ISSUE TITLE
.gitignore backups and restore backups from separate directory?

### ISSUE THREAD

**Issue by nickautomatic** [niceandserious/crafty-vagrant#7]
Two problem with having backups in Git:

1. Potential clashes with untracked files when pulling in on production server after downloading backup
2. Repo size inflation

1 is a more serious problem. Occurred to me that if the db restore script didn't use Craft's own "backups" directory then it could be gitignored (and potentially backups could be stored as zips rather than raw SQL? In which case .sql files could be ignored and the script could restore the latest zip)

**Comment by chrisdubya** [comment]
Agreed, craft says you should ignore as well [here](https://craftcms.com/support/craft-storage-gitignore).
